concepts/Common/Integration Guides/20 Create a DevExtreme application with Next.js/20 Create a DevExtreme application with Next.js.md

```diff
@@ -0,0 +1,208 @@
+# Create a DevExtreme application with Next.js
+
+Next.js is a React-based web development framework. It can generate static websites or, with a little extra configuration, serve dynamic content. The creators of React endorse Next.js because Next is easy to use, feature-rich, and compatible with modern web development patterns, such as server-side page rendering.
+
+This article explains how to create a simple Next.js application, and populate it with DevExtreme components.
+
+- [1. Create a new project](#1-create-a-new-project)
+- [2. Add static content to the front page](#2-add-static-content-to-the-front-page)
+  - [Clean-up](#clean-up)
+  - [View the server-rendered page](#view-the-server-rendered-page)
+- [3. Add a DevExtreme DataGrid](#3-add-a-devextreme-datagrid)
+  - [Install dependencies](#install-dependencies)
+  - [Set up the necessary imports](#set-up-the-necessary-imports)
+  - [Set up a Data Source](#set-up-a-data-source)
+  - [Configure the DataGrid](#configure-the-datagrid)
+  - [Insert the DataGrid component into the page](#insert-the-datagrid-component-into-the-page)
+  - [View the client-side rendered grid](#view-the-client-side-rendered-grid)
+- [4. Next Steps](4-next-steps)
+
+## 1. Create a new project
+
+> Read the [official Next.js documentation](https://nextjs.org/docs/getting-started/installation) for an in-depth overview of the installation process.
+
+The `create-next-app` wizard allows you to bootstrap a Next.js web app in a matter of minutes.
+
+![create-next-app](/images/NextJS/create-next-app.png)
+
+The wizard creates a folder for the project, installs the necessary dependencies, and populates the project with sample code.
+
+Run the following command to launch the wizard:
+
+    npx create-next-app@latest
+
+Answer the wizard's prompts to configure your new app. For the purposes of this tutorial, we enabled the following options:
+
+* TypeScript support
+* ESLint support
+* Store source code in the `src/` directory
+* Use the Next.js App Router
+
+After the wizard completes its job, you can start the development server:
+
+    npm run dev
+
+If you want to run your website in **production mode**, build the project first:
+
+    npm run build; npm run start
+
+The server outputs the preview URL to the console. The default URL is `http://localhost:3000`. Open the URL in the browser to view the page.
+
+![Original front page](/images/NextJS/original-front-page.png)
+
+Proceed to the next step when you get the server up and running.
+
+## 2. Add static content to the front page
+
+Next.js stores the source code for your application's front page in the `src/app/page.tsx` file.
```

Comment:
Only true if you specifically choose to create a `src` folder. This is a question asked during the creation process, but the default is now "no".